lib/modules/dosomething/dosomething_rogue/includes/Rogue.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+use DoSomething\Gateway\Common\RestApiClient;
+use DoSomething\Gateway\ForwardsTransactionIds;
+
+class Rogue extends RestApiClient {
+  use ForwardsTransactionIds;
+
+  /**
+   * The class name of the transaction framework bridge.
+   *
+   * @var string
+   */
+  protected $transactionBridge = Phoenix\Auth\PhoenixTransactionBridge::class;
+
+  /**
+   * Create a new Rogue API client.
+   */
+  // public function __construct() {
+  //   $url = ROGUE_API_URL . '/' . ROGUE_API_VERSION . '/';
+
+  //   parent::__construct($url);
+  // }
+
+
+  /**
+   * Send a POST request of the reportback to be saved in Rogue.
+   *
+   * @param string $baseurl
+   * @param array $data
+   * @return object|false
+   */
+  public function postReportback($baseurl, $data) {
+    $response = $this->post(ROGUE_API_URL . '/' . ROGUE_API_VERSION . '/' . $baseurl, $data);
```

Comment:
Also, when I `var_dump($url); die():` [above this line](https://github.com/DoSomething/gateway/blob/master/src/Common/RestApiClient.php#L44), it also is away of the `$url`: 
```
/var/www/dev.dosomething.org/vendor/dosomething/gateway/src/Common/RestApiClient.php:44:string 'http://rogue.dev:8000/api/v1' (length=28)
```